dynawo/sources/Modeler/Common/DYNModel.h

```diff
@@ -422,6 +422,21 @@ class Model {
    */
   virtual void loadVariables(const std::map< std::string, std::string>& mapVariables) = 0;
 
+  /**
+   * @brief export the internal variables of the model for dump
+   *
+   * @param mapInternalVariables : map associating the file where values should be dumped with the stream of values
+   */
+  virtual void dumpInternalVariables(std::map< std::string, std::string >& mapInternalVariables) = 0;
```

Comment:
No need to have this visible at that level. Specializing the dumpVariables of components of the model network should be enough